### ISSUE TITLE
problems in two Submission Sheets

### ISSUE THREAD

**Issue by agarwal** [cgsb/hitscore#97]
Below are issues from an old Google Doc, before we started using GitHub's Issue system. Both issues are not important to resolve, so closing immediately. But addding issue for our records.

2011-12-05 AD03L7ACXX L2-L7 Rushlow
- Missing bioanalyzer info for one library. Shengbo said he never ran bioanalyzer on this one. He might do this at some point, but it is not a priority.

2011-12-05 BD03NWACXX L7 Scicchitano
- Min/Max Fragment Sizes missing, sent email asking for investigators to fill out this info (PES: this was an amplicon-seq project, so their is no size range). This means min/max are equal to mean according to Paul, but John Burns said the amplicons had more than one band by design, so again not sure what values to enter.
  Awaiting Paul’s reply.
- Protocol is "PCR". Paul has asked investigators for clarification. (PES: John Burns sent email saying he is working on this).
